src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -76,6 +80,7 @@ export class ViewConfigItemComponent implements OnInit, OnChanges {
       return this.configItems.map(conf => {
         if (conf.key === d.key) {
           d['type'] = conf.type;
+          d.value = d.value.toString();
```

Comment:
why type in `[]` and value with `.`

Reply:
there is no key name `type ` in the object, so to put it there we need to define it.

Reply:
Okay, please add a comment `// there is no key 'type' in the object` before the line `d['type']`